scene/2d/animated_sprite_2d.h

```diff
@@ -38,14 +38,9 @@ class SpriteFrames : public Resource {
 	GDCLASS(SpriteFrames, Resource);
 
 	struct Anim {
-		float speed;
-		bool loop;
+		float speed = 5;
```

Comment:
```suggestion
		float speed = 5.0;
```
Just to make the `float` explicit while we're at it.